doc/man7/openssl-env.pod

```diff
@@ -0,0 +1,78 @@
+=pod
+
+=head1 NAME
+
+openssl-env - OpenSSL environment variables
+
+=head1 DESCRIPTION
+
+The OpenSSL libraries use environment variables to override the
+compiled-in default paths for various data.
+To avoid security risks, the environment is usually not consulted when
+the executable is set-user-ID or set-group-ID.
+
+=over 4
+
+=item B<CTLOG_FILE>
+
+Specifies the path to a certificate transparency log list.
+See L<CTLOG_STORE_new(3)>.
+
+=item B<OPENSSL>
+
+Specifies the path to the B<openssl> executable. Only used by
+the B<rehash> script.
```

Comment:
If it's specific to just one or maybe two commands, I don't see a reason to move the description here.

Reply:
having all environment variables in one place is worthwhile for security reasons.  I would rather not remove it from here unless there's strong objection.

Reply:
I think it's rather piss poor documentation if, when I am interested in `openssl rehash`, to have to jump to another manual page just to find out that this environment variable is unique for the command I was looking up in the first place.

I was thinking of this page to hold the *common* environment variables (most of all, those that the libraries look at, i.e that are even relevant for other applications that link with our libraries), not *all* environment variables. That's probably the reason my PR is relatively terse in comparison.

Reply:
The rehash manpage *is not changed.*